pkgs/jnigen/lib/src/bindings/excluder.dart

```diff
@@ -40,23 +43,30 @@ class _ClassExcluder extends Visitor<ClassDecl, void> {
   @override
   void visit(ClassDecl node) {
     node.methods = node.methods.where((method) {
-      final included = !_isPrivate(method) &&
-          !method.name.startsWith('_') &&
-          method.name != '<clinit>' &&
-          (config.exclude?.methods?.included(node, method) ?? true);
-      if (!included) {
+      final isPrivate = method.isPrivate;
+      final hasPrivateName = method.hasPrivateName;
+      final isAbstractCtor = method.isCtor && node.isAbstract;
+      final isBridgeMethod = method.isSynthetic && method.isBridge;
+      final isExcludedInConfig =
+          config.exclude?.methods?.included(node, method) ?? false;
+      final excluded = isPrivate ||
+          hasPrivateName ||
```

Comment:
A leading underscore doesn't mean anything in Java. Are you excluding the method purely because it would be private when generated in Dart? Would it be better to rename the method? I guess there probably aren't many methods that hit this case in practice.

Reply:
Yes. I need to come up with a default renaming scheme to include those.

Reply:
Will do in a separate PR https://github.com/dart-lang/native/issues/1523